data/json/items/comestibles/dairy.json

```diff
@@ -242,6 +242,22 @@
     "fun": 7,
     "vitamins": [ [ "calcium", 16 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "cottage_cheese",
+    "name": { "str_sp": "cottage cheese" },
+    "copy-from": "cheese",
+    "weight": "34 g",
+    "spoils_in": "180 days",
+    "quench": 1,
+    "calories": 391,
+    "description": "Hard, dry cheese made to last, unlike modern processed cheese.  Will make you thirsty though.",
```

Comment:
Sorry I went to bed halfway through and didn't rewrite most of the things I copied. And have since been absolutely without any free time after that moment